checkov/terraform/checks/resource/aws/S3AllowsAnyPrincipal.py

```diff
@@ -0,0 +1,40 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+import json
+
+
+class S3AllowsAnyPrincipal(BaseResourceCheck):
+
+    def __init__(self):
+        name = "Ensure S3 bucket does not allow an action with any Principal"
+        id = "CKV_AWS_70"
+        supported_resources = ['aws_s3_bucket', 'aws_s3_bucket_policy']
+        categories = [CheckCategories.IAM]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if isinstance(conf['policy'][0], str):
+            try:
+                policy_block = json.loads(conf['policy'][0])
+                if 'Statement' in policy_block.keys():
```

Comment:
`if 'Statement' in policy_block:` is enough. But this is still correct.

I just ask to gather some opinions about this case. Calling keys is not necessary but in some way more expressive. On the other hand asking if something is in a dict is also really clear.